packages/upset/package.json

```diff
@@ -67,8 +67,10 @@
     "@mui/material": "^5.11.0",
     "@mui/system": "^5.11.0",
     "@mui/x-data-grid": "^5.17.12",
-    "@visdesignlab/trrack": "^2.0.0-alpha.10",
-    "@visdesignlab/trrack-vis": "^2.0.0-alpha.12",
+    "@reduxjs/toolkit": "^1.9.1",
+    "@swc/core": "^1.3.29",
+    "@swc/helpers": "^0.4.14",
```

Comment:
Let's open an issue to remind us that we need to remove these when Kiran has updated the packaging for trrack